project.Makefile

```diff
@@ -131,45 +131,32 @@ make-rdf: rdf-clean \
 # todo: metagenome_sequencing_set and metagenome_sequencing_activity_set are degenerate
 #   and can't be validated, migrated or converted to RDF
 
+# --selected-collections workflow_execution_set
+
 local/mongo_as_unvalidated_nmdc_database.yaml:
 	date
 	time $(RUN) pure-export \
 		--max-docs-per-coll 200000 \
 		--output-yaml $@ \
 		--schema-source src/schema/nmdc.yaml \
-		--selected-collections activity_set \
 		--selected-collections biosample_set \
+		--selected-collections calibration_set \
+		--selected-collections chemical_entity_set \
 		--selected-collections collecting_biosamples_from_site_set \
+		--selected-collections configuration_set \
+		--selected-collections data_generation_set \
 		--selected-collections data_object_set \
-		--selected-collections extraction_set \
 		--selected-collections field_research_site_set \
 		--selected-collections functional_annotation_set \
 		--selected-collections genome_feature_set \
-		--selected-collections library_preparation_set \
-		--selected-collections mags_activity_set \
-		--selected-collections mags_set \
-		--selected-collections material_sample_set \
-		--selected-collections metabolomics_analysis_activity_set \
-		--selected-collections metabolomics_analysis_set \
-		--selected-collections metagenome_annotation_activity_set \
-		--selected-collections metagenome_annotation_set \
-		--selected-collections metagenome_assembly_set \
-		--selected-collections metagenome_sequencing_activity_set \
-		--selected-collections metagenome_sequencing_set \
-		--selected-collections metap_gene_function_aggregation \
-		--selected-collections metatranscriptome_activity_set \
-		--selected-collections metatranscriptome_analysis_set \
-		--selected-collections nom_analysis_activity_set \
-		--selected-collections omics_processing_set \
-		--selected-collections planned_process_set \
-		--selected-collections pooling_set \
+		--selected-collections instrument_set \
+		--selected-collections material_processing_set \
 		--selected-collections processed_sample_set \
-		--selected-collections read_based_taxonomy_analysis_activity_set \
-		--selected-collections read_qc_analysis_activity_set \
+		--selected-collections protocol_execution_set \
+		--selected-collections storage_process_set \
 		--selected-collections study_set \
-		--selected-collections workflow_execution_set \
 		dump-from-api \
-		--client-base-url "https://api.microbiomedata.org" \
+		--client-base-url "https://api-berkeley.microbiomedata.org" \
```

Comment:
should prod code point to berkeley?

Reply:
yes, starting Monday of next week it should